reference/forms/types/entity.rst

```diff
@@ -202,9 +202,10 @@ query_builder
 Allows you to create a custom query for your choices. See
 :ref:`ref-form-entity-query-builder` for an example.
 
-The value of this option can either be a ``QueryBuilder`` object, a Closure or null.
+The value of this option can either be a ``QueryBuilder`` object, a Closure or ``null``.
 When using a Closure, you will be passed the ``EntityRepository`` of the entity
-as the only argument and should return a ``QueryBuilder``.  If you'd like to display a list of empty entries, you can return null in the query_builder closure.
+as the only argument and should return a ``QueryBuilder``.  
+If you'd like to display a list of empty entries, you can return ``null`` in the closure.
```

Comment:
Looks like there was something wrong with the commit and it was fixed in https://github.com/symfony/symfony/commit/eb08baa9fcb6db3863c795ea54e324ced68d56ea

The tests exists and passes: https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Doctrine/Tests/Form/Type/EntityTypeTest.php#L212-L223